custom-eureka-client/src/main/java/com/example/springboot/AccessTokenManager.java

```diff
@@ -0,0 +1,34 @@
+package com.example.springboot;

```

Comment:
Add license header for all Java files like below.

https://github.com/Azure-Samples/Azure-Spring-Cloud-Samples/blob/3befb6da96cec2ffb6bd34661c16bc6d8f2108b5/java-8-sample/src/main/java/com/microsoft/sample/java8ASCdemo/Java8AscDemoApplication.java#L1-L5

Reply:
updated.